docs/content/sidebars/ts-sdk.js

```diff
@@ -125,11 +126,20 @@ const tsSDK = [
     {
         type: 'category',
         label: 'API',
-        link: {
-            type: 'doc',
-            id: 'ts-sdk/api/index',
-        },
-        items: typedocSidebar,
+        items: [
+            {
+                type: 'category',
+                label: 'Testnet',
+                items: typedocSidebarTestnet,
+                link: { type: 'doc', id: 'ts-sdk/api/index' },
```

Comment:
It would have been more friendly if the id had reflected `testnet` just as the `devnet` section, except if this is intentional for now, and will be updated.